net/net.sh

```diff
@@ -948,6 +950,20 @@ while [[ -n $1 ]]; do
     elif [[ $1 = --skip-require-tower ]]; then
       maybeSkipRequireTower="$1"
       shift 1
+    elif [[ $1 = --client-type ]]; then
+      clientType=$2
+      case "$clientType" in
+        thin-client|tpu-client|rpc-client)
+          ;;
+        *)
+          echo "Unexpected client type: \"$clientType\""
+          exit 1
+          ;;
+      esac
+      shift 2
+    elif [[ $1 = --url ]]; then
+      url=$2
+      shift 2
```

Comment:
Remove this. We can re-add an override if it seems problematic in practice to have rpc-/tpu-clients always using the bootstrap node's RPC service.